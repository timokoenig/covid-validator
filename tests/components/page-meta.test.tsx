import React from 'react'
import renderer from 'react-test-renderer'
import PageMeta from '../../components/page-meta'

it('renders correctly', () => {
  const tree = renderer.create(<PageMeta />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders correctly without index', () => {
  const tree = renderer.create(<PageMeta allowIndex={false} />).toJSON()
  expect(tree).toMatchSnapshot()
})
