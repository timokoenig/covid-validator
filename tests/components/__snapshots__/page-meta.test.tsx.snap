// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `null`;

exports[`renders correctly without index 1`] = `null`;
